src/vivarium_public_health/disease/state.py

```diff
@@ -196,17 +196,11 @@ def setup(self, builder):
             'prevalence', lambda cause, builder: builder.data.load(f"{self.cause_type}.{cause}.prevalence"))
         get_dwell_time_func = self._get_data_functions.get('dwell_time', lambda *args, **kwargs: pd.Timedelta(0))
 
-        disability_weight_data = get_disability_weight_func(self.cause, builder)
         self.prevalence_data = builder.lookup.build_table(get_prevalence_func(self.cause, builder))
         self._dwell_time = get_dwell_time_func(self.cause, builder)
         self.randomness_prevalence = builder.randomness.get_stream(f'{self.state_id}_prevalent_cases')
 
-        if isinstance(disability_weight_data, pd.DataFrame):
```

Comment:
@kfwilson the single value was breaking my tests and I thought it was because of this change. I'll check agian

Reply:
weird. I would have thought it would be fine since it gets passed to a lookup table